### ISSUE TITLE
Include slim version

### ISSUE THREAD

**Issue by benjavalero** [webjars/jquery#72]
Should the **slim** version of jQuery be included in this package, or in a dedicated one? I don't know well if there is a real benefit in using it, but it is recommended for instance by Bootstrap.

**Comment by mgimpel** [comment]
I adding this to the POM file would seem to do the job
```
                    <execution>
                        <id>download-slim-js</id>
                        <phase>process-resources</phase>
                        <goals><goal>download-single</goal></goals>
                        <configuration>
                            <url>${downloadUrl}</url>
                            <fromFile>jquery.slim.js</fromFile>
                            <toFile>${destinationDir}/jquery.slim.js</toFile>
                        </configuration>
                    </execution>
                    <execution>
                        <id>download-slim-min-js</id>
                        <phase>process-resources</phase>
                        <goals><goal>download-single</goal></goals>
                        <configuration>
                            <url>${downloadUrl}</url>
                            <fromFile>jquery.slim.min.js</fromFile>
                            <toFile>${destinationDir}/jquery.slim.min.js</toFile>
                        </configuration>
                    </execution>
                    <execution>
                        <id>download-slim-source-map</id>
                        <phase>process-resources</phase>
                        <goals><goal>download-single</goal></goals>
                        <configuration>
                            <url>${downloadUrl}</url>
                            <fromFile>jquery.slim.min.map</fromFile>
                            <toFile>${destinationDir}/jquery.slim.min.map</toFile>
                        </configuration>
                    </execution>
```

**Comment by jamesward** [comment]
Thanks!  This is being released as `3.3.1-2`
